app/views/mail_form/contact.erb

```diff
@@ -13,10 +13,6 @@
   %></p>
 <% end %>
 
-<% if @resource.request_id.present? %>
-  <p><b>Request ID:</b> <%= @resource.request_id %></p>
-<% end %>
```

Comment:
Where did this go?

Reply:
In the model, I made `request_id` to `attribute`, so it will get added to the email automatically (because all the attributes are appended in this file above).